net-libs/libbitcoinconsensus/libbitcoinconsensus-0.14.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2010-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+BITCOINCORE_COMMITHASH="964a185cc83af34587194a6ecda3ed9cf6b49263"
+BITCOINCORE_LJR_DATE="20170420"
+BITCOINCORE_IUSE=""
+BITCOINCORE_NEED_LIBSECP256K1=1
+BITCOINCORE_NO_DEPEND="libevent"
+inherit bitcoincore eutils
+
+DESCRIPTION="Bitcoin Core consensus library"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux ~arm ~arm64 ~mips ~ppc ~x86 ~x86-linux"
+
+src_configure() {
+	bitcoincore_conf \
+		--with-libs
+}
+
+src_install() {
+	bitcoincore_src_install
+	dodoc doc/bips.md
```

Comment:
below `KEYWORDS`, declare a `DOCS` array as follows:
```
DOCS=( doc/bips.md )
```
You can then kill this line.